python/src/aac/lang/__init__.py

```diff
@@ -1 +1,9 @@
 """A module that provides AaC language features."""
+
+from ._active_context import ActiveContext
+from .context_manager import get_active_context
```

Comment:
I think if we expose a submodule in a package, then the submodule should be made private and treated as such.